manifests/v/venko/ClassicGenerator/5.0.0.3/venko.ClassicGenerator.locale.en-US.yaml

```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: venko.ClassicGenerator
+PackageVersion: 5.0.0.3
+PackageLocale: en-US
+Publisher: venko
+PackageName: Classic Generator
+License: MPL-2.0 license
+ShortDescription: Classic Generator Setup

```

Comment:
Please provide a better description of what 'Classic Generator' as a program does